### ISSUE TITLE
Share the mean and var of batch norm when Multi gpus training

### ISSUE THREAD

**Issue by LZDSJTU** [tensorflow/tensorflow#21311]
I want to know if using estimator to train on multi gpus can share the mean and var of batchnorm.

As the example in cifar, it cannot deal with the batchnorm sharing between different gpu

**Comment by ppwwyyxx** [comment]
Duplicate of #18222
